### DISCUSSION TITLE
ISSUE

### DISCUSSION THREAD

**Discussion by FunkyEgg** [DigitalLogicSimCommunity/Digital-Logic-Sim--Unofficial#4] [Q&A]
when you hide custom chips it hides all chips

**Comment by AOx0** [comment]
Does this still happen?

**Reply by FunkyEgg** [reply]
> Does this still happen? 

I'm assuming so since I see no new release which means no new compile, the issue at hand is that it hides all custom chips not just the default new custom

**Reply by FunkyEgg** [reply]
Don't worry though since I have switched programs

**Reply by AOx0** [reply]
From v0.35 to v0.34 or what do you mean?
